src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -143,6 +149,16 @@ public boolean doneScoring() {
   public void initialize() {
     armEncoder.setPosition(
         armAbsoluteEncoder.getPosition() + Calibrations.ARM_ABSOLUTE_ENCODER_OFFSET_DEG);
+
+    double elevatorDutyCycleEncodersDifference =
```

Comment:
Add units to variable name, also can we do `(a - b) * degPerRev` instead of `a * degPerRev - b * degPerRev`, it's just a little cleaner